composer.json

```
@@ -6,27 +6,23 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/cas": "1.6",
-        "drupal/core": "^8.9",
-        "php": ">=7.2"
+        "php": ">=7.3",
+        "drupal/core": "^8.9 || ^9.1",
+        "drupal/cas": "^1.7"
     },
     "require-dev": {
-        "composer/installers": "~1.5",
-        "drupal/cas_mock_server": "^1.0@alpha",
-        "drupal/core-composer-scaffold": "^8.9",
+        "composer/installers": "^1.11",
```

Better to keep previous constraint approach:
```suggestion
        "composer/installers": "~1.11",
```